l10n_it_fatturapa_pec/tests/test_e_invoice_response.py

```diff
@@ -2,9 +2,9 @@
 # Copyright 2018 Simone Rubino - Agile Business Group
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment:
Andrebbe modificato come sopra

Reply:
Grazie della review! In effetti le licenze del modulo erano un po' mischiate, le ho messe tutte AGPL